src/main/java/com/example/medical_clinic_proxy/controller/VisitController.java

```diff
@@ -0,0 +1,39 @@
+package com.example.medical_clinic_proxy.controller;
+
+import com.example.medical_clinic_proxy.model.PageContent;
+import com.example.medical_clinic_proxy.model.dto.VisitDTO;
+import com.example.medical_clinic_proxy.service.VisitService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequestMapping("/visits")
+@RequiredArgsConstructor
+public class VisitController {
+    private final VisitService visitService;
+
+    @GetMapping("/{patientEmail}")
```

Comment:
localhost:8080/visits/kg@gmail.com <- ten email to nie jest defakto cześć wizyty to nie jest wartośc unikalna w kontekście wizyty -> to jest wartośc unikalna w kontekście Pacjenta. 
Więc to co my tutaj robimy to my chcemy tak naprawde przefiltrować zbiór wizyt o wizyty danego pacjenta -> 
requestParam zamiast pathVariable

Reply:
localhost:8080/visits?patientEmail=kg@gmail.com